crates/orderbook/openapi.yml

```diff
@@ -540,32 +530,113 @@ components:
         - partiallyFillable
     OrderCreation:
       description: Data a user provides when creating a new order.
-      allOf:
-        - $ref: "#/components/schemas/OrderParameters"
-        - type: object
-          properties:
-            signingScheme:
-              $ref: "#/components/schemas/SigningScheme"
-            signature:
-              $ref: "#/components/schemas/Signature"
-            from:
-              description: |
-                If set, the backend enforces that this address matches what is decoded as the signer of
-                the signature. This helps catch errors with invalid signature encodings as the backend
-                might otherwise silently work with an unexpected address that for example does not have
-                any balance.
-              allOf:
-                - $ref: "#/components/schemas/Address"
-              nullable: true
-            quoteId:
+      type: object
+      properties:
```

Comment:
this adds a bit of repetition. Is there a way to keep it more DRY? (or is this being auto-generated?)

Reply:
The only way to not repeat is to create a new object that contains all the fields that are shared between `OrderCreation` and `OrderData` and referencing that object from both use cases. I didn't do that because it didn't feel like the right semantics. It would only exist to hold the shared fields without modelling anything. Not sure I can explain this right.

Reply:
it's just so you don't need to C&P descriptions, examples, etc. Keeping it DRY will reduce the errors for example if you do one change in one field but don't do it in another. They are extremely similar. 

>  It would only exist to hold the shared fields without modelling anything. Not sure I can explain this right.

That wouldn't be a problem, it's just an abstraction that refactor both types. 

Anyways,is not that important to me, I just mentioned because this might be more error prone.